framework/src/modules/token/schemas.ts

```diff
@@ -46,6 +46,10 @@ export const configSchema = {
 				format: 'hex',
 			},
 		},
+		accountInitializationFee: {
+			type: 'string',
+			format: 'uint64',
+		},
```

Comment:
```suggestion
		accountInitializationFee: {
			type: 'string',
			format: 'uint64',
		},
		feeTokenID: {
			type: 'string',
			format: 'hex',
			minLength: TOKEN_ID_LENGTH,
			maxLength: TOKEN_ID_LENGTH,
		},
```
I think i forgot to add the feeTokenID 🙏